src/ragger/backend/speculos.py

```diff
@@ -74,6 +74,7 @@ def __init__(self,
                                                       api_url=self.url,
                                                       **kwargs)
         self._pending: Optional[ApduResponse] = None
+        self._screen_content: dict = {}
```

Comment:
@xchapron-ledger so now we require the context to be some kind of dict ? (and not "Any")

Reply:
Oh I see, since it's the backend and not the navigator we can assume it will be a dict. Disregard my comment.

Reply:
It's for something not exposed.
On the `interface` it's still declared as `Any`, but in the internal of `speculos` backend we known that in this case it is a `Dict`.